scipy/ndimage/_filters.py

```diff
@@ -409,6 +420,28 @@ def prewitt(input, axis=-1, output=None, mode="reflect", cval=0.0):
     >>> ax1.imshow(ascent)
     >>> ax2.imshow(result)
     >>> plt.show()
+
+    In the example above, the directional filter is returned,
```

Comment:
We are effectively showing the same thing twice here, I think we can just follow the lead of sobol and just show the four image grid